dev-haskell/ghc-lib-parser/ghc-lib-parser-8.10.2.20200916.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
Same thing, I thought this was added in your hlint PR. Perhaps need a rebase against master?

Reply:
Yes definitely need to rebase against latest master.